models/gif_psc_exp_multisynapse.h

```diff
@@ -0,0 +1,422 @@
+/*
+ *  gif_psc_exp_multisynapse.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef GIF_PSC_EXP_MULTISYNAPSE_H
+#define GIF_PSC_EXP_MULTISYNAPSE_H
+
+// Includes from nestkernel:
+#include "archiving_node.h"
+#include "ring_buffer.h"
+#include "connection.h"
+#include "event.h"
+#include "universal_data_logger.h"
+
+#include "nest.h"
+
+/* BeginDocumentation
+  Name: gif_psc_exp_multisynapse - Current based generalized integrate-and-fire
+  neuron model according to Mensi et al. (2012) and Pozzorini et al. (2015).
```

Comment:
"Current-based". Also, the short description is identical to `gif_psc_exp`. Maybe remove the references but add "with multiple synaptic time constants."